src/main/java/world/anhgelus/molehunt/Molehunt.java

```diff
@@ -23,6 +34,50 @@ public class Molehunt implements ModInitializer {
 
     public static final String MOD_ID = "molehunt";
     public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);
+    public static Config CONFIG;
+
+    public static final SimpleConfig CONFIG_FILE = Config.configFile(MOD_ID);
+
+    public static final GameRules.Key<GameRules.IntRule> GAME_DURATION = GameRuleRegistry.register(
+            MOD_ID +":gameDuration",
+            GameRules.Category.MISC,
+            GameRuleFactory.createIntRule(CONFIG_FILE.getOrDefault("game_duration", 90))
```

Comment:
It would be better for the default config values to be in the Config class, for easier access if they need to be changed (even if it's unlikely)

Reply:
I didn't want to create two classes for these two configs because it's overkill

Reply:
You don't need another class, just add fields to the Config class :
```java
public static int DEFAULT_GAME_DURATION = 90;
// etc.
```
and use them here instead of putting the values directly.

Reply:
We only use them in one place, it's overkill

Reply:
It's better because then they're all together in the same place

Reply:
They are already in the same place